cmd/graph_demo/main.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"math/rand"
+	"time"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/app"
+	"fyne.io/fyne/v2/container"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+	"fyne.io/x/fyne/widget/charts"
+)
+
+func main() {
+	app := app.New()
+	w := app.NewWindow("Graphs")
+
+	graphWidgets := make([]fyne.CanvasObject, 0)
```

Comment:
```suggestion
	graphWidgets := []fyne.CanvasObject{}
```
Or perhaps this one?
```suggestion
	var graphWidgets []fyne.CanvasObject
```
I don't think you necessarily need to use make to create an empty slice.

Reply:
The second one is probably better. It requires no allocations until the actual `append()` call. There's no reason to use RAM before it's needed, especially when the change is so simple.

Reply:
It might also be worth considering using `make()` here and setting the capacity because the amount of elements seems to be known, and seems to always be 6 in this case, so `make([]fyne.CanvasObject, 0, 6)`. That way, it will only require one allocation and no copying. Without this, it is likely that `append()` will end up having to allocate a new slice, and copy all the elements over to the new slice multiple times, and it would also likely end up using more memory if `append()` allocates more than required (which it probably will).

Overall, this isn't really a big deal, but it's nice not to use unneeded resources and time for no reason.